migrations/20190508141011-create-article.js

```diff
@@ -0,0 +1,27 @@
+const articleMigration = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Articles', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    title: {
+      type: Sequelize.STRING
+    },
+    user_id: {
```

Comment:
this attribute should be in camelCase. but I would like to suggest to use the association file migration created by @Niyitangasam because it will help you create this attribute while migrating. furthermore, it is already merged on develop so it will help you reduce conflicts.